EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -1206,297 +1206,297 @@ QUEST_LV_0300_20160310_001205	What do we do with the useless scriptures?{nl}We u
-QUEST_LV_0300_20160310_001420	Now, everything is screwed up.{nl}Picking up corals went bad and our village is finished..
-QUEST_LV_0300_20160310_001421	It was suspicious.{nl}We can't move the soldiers if it's not tributes, does it make sense?{nl}It is so fortunate that it was solved well because of you.
-QUEST_LV_0300_20160310_001422	Even though we did say we would pay the cost on behalf of the townspeople but the soldiers would only move for the offerings.{nl}Without any means to gather corals, from where can the offerings be produced?{nl}Those poor townspeople.. They can never catch their breaks.{nl}
-QUEST_LV_0300_20160310_001423	Damn..{nl}We should get the corals quickly and go back..
-QUEST_LV_0300_20160310_001424	I told them that there are many monsters, but they still went to the workplace and got hurt.{nl}We are so desperatem but it seems tributes are more important for the agent than people.
-QUEST_LV_0300_20160310_001425	This pocket was buried in the spot where the odor was coming from?{nl}Let's see..
-QUEST_LV_0300_20160310_001426	The number of monsters increased a lot so the number of wounded people is increasing.{nl}We don't have enough medicine..
-QUEST_LV_0300_20160310_001427	If it weren't you..
-QUEST_LV_0300_20160310_001428	I knew it was strange.{nl}It was an order which we couldn't understand.
-QUEST_LV_0300_20160310_001429	It is difficult.{nl}The deadline is coming, but we can't get the corals..
+QUEST_LV_0300_20160310_001420	Now, everything is screwed up.{nl}Picking up corals went bad and our village is finished for...
+QUEST_LV_0300_20160310_001421	It was suspicious.{nl}We can't move the soldiers if it's not tributes, does it make any sense?{nl}It's good that it was solved because of you.
+QUEST_LV_0300_20160310_001422	Even though we did say we would pay the cost on behalf of the {nl}townspeople but the soldiers would only move for the tributes.{nl}Without any means to gather corals, from where can the tributes be produced?{nl}Those poor townspeople... They can never catch their breaks.{nl}
+QUEST_LV_0300_20160310_001423	What should we do...?{nl}We should get the corals fast and go back...
+QUEST_LV_0300_20160310_001424	I told them that there are many monsters, but they still went to the workplace and got hurt.{nl}We are so desperate but it seems the tributes are more important for the lieutenant than people.
+QUEST_LV_0300_20160310_001425	This pocket was buried in the spot where the smell was coming from?{nl}Let's see...
+QUEST_LV_0300_20160310_001426	The number of monsters increased a lot so the number of wounded people is increasing.{nl}We don't have enough medicine...
+QUEST_LV_0300_20160310_001427	If it weren't you...
+QUEST_LV_0300_20160310_001428	I knew it was strange.{nl}It was a smell we couldn't understand.
+QUEST_LV_0300_20160310_001429	It is difficult.{nl}The deadline is coming, but we can't grab the corals...
```

Comment:
"These are hard times.{nl}The deadline is coming up, but we can't get any coral..."